front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -115,7 +128,7 @@ const ScenarioDescription = ({
       </div>
       <div className="scenario-details-electrical-profile-set">
         {scenario.electrical_profile_set_id
-          ? scenario.electrical_profile_set_id
+          ? electricalProfileSet?.name || ''
```

Comment:
Is it possible that an electrical profile has no name? In which case, will we ever get the || ' ' case ?

Reply:
Done :white_check_mark:

Reply:
Ty ! LGTM